trk/src/Screens/Home/index.js

```diff
@@ -48,44 +48,24 @@ function HomeScreen(props) {
     if (Platform.OS === 'android') {
       androidPromptRef.current.setVisible(true);
     }
-
+  
     try {
       await NfcManager.requestTechnology(NfcTech.NfcA);
-      const { getClimb } = ClimbsApi();
-      const climbId = await readClimb();
-      const climbData = await getClimb(climbId[0]); // Fetch climb data using ID
-      if (climbData.exists) { //check if climb exists and if the user is the setter, if not, allow them to read the climb
-        if (currentUser.uid !== climbData.data().setter) {
-          const { addTap } = TapsApi();
-          const tap = {
-            climb: climbId[0],
-            user: currentUser.uid,
-            timestamp: new Date(),
-            completion: 0,
-            attempts: '',
-            witness1: '',
-            witness2: '',
-          }
-          const documentReference = await addTap(tap);
-          const newTapId = documentReference.id;
-          navigation.navigate('Detail', { climbData: climbData.data(), tapId: newTapId, climbId: climbId[0] });
-        } else {
-          navigation.navigate('Detail', { climbData: climbData.data(), climbId: climbId[0] })
-        }
-
```

Comment:
this has now all been moved to climbdetail - the only thing that blocks navigation is an unsuccessful nfc read